pkg/sql/ttl/ttlschedule/ttlschedule.go

```diff
@@ -217,12 +217,18 @@ func createRowLevelTTLJob(
 	ctx context.Context,
 	createdByInfo *jobs.CreatedByInfo,
 	txn *kv.Txn,
-	ie sqlutil.InternalExecutor,
+	descsCol *descs.Collection,
 	jobRegistry *jobs.Registry,
 	ttlDetails catpb.ScheduledRowLevelTTLArgs,
 ) (jobspb.JobID, error) {
+	f := tree.NewFmtCtx(tree.FmtSimple)
+	tn, err := descs.GetTableNameByID(ctx, txn, descsCol, ttlDetails.TableID)
+	if err != nil {
+		return 0, err
+	}
+	f.FormatNode(tn)
```

Comment:
just noticed that this is kinda duplicating the logic of just calling `tn.String()` (or `tree.AsString(tn)`). just a nit, not a big deal

Reply:
ah! fixed.